src/FacultyDirectory/Controllers/SystemController.cs

```diff
@@ -105,6 +105,31 @@ public async Task<ActionResult> PeopleWithSources()
             return Json(result);
         }
 
+        [HttpGet]
+        public async Task<ActionResult> CreateSitePeople(int id)
+        {
+            var validSite = await this.dbContext.Sites.AnyAsync(s => s.Id == id);
+
+            if (!validSite) {
+                return NotFound();
+            }
```

Comment:
@srkirkland Maybe only run if sitePerson is empty?

Reply:
if the whole table is you mean?